charts/tfy-llm-gateway-infra/values.yaml

```diff
@@ -34,6 +34,12 @@ global:
       ## clickhouse password
       ##
       passwordKey: "CLICKHOUSE_USER_PASSWORD"
+  ## @param global.labels [object] labels
+  ##
+  labels: {}
+  ## @param global.annotations [object] annotations
+  ##
+  annotations: {}
```

Comment:
Let's add support for nats.labels and nats.annotations also in the values.yaml so that we can reference them in the helpers.tpl